tests/unit/components/textInput.test.js

```diff
@@ -0,0 +1,163 @@
+import {render, unmountComponentAtNode} from 'react-dom';
+import React from 'react';
+import {act} from 'react-dom/test-utils';
+import TextInput from '../../../compoments/textInput';
+
+let container = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+describe('textInput', () => {
+  test('Displays title', () => {
+    const titleText = 'A Title'
+    const textInputName= 'textInputName';
```

Comment:
I'm wondering if it's worth putting `const textInputName= 'textInputName';` above the tests to make it accessible to all the tests & then we won't need to repeat it.